RiotNSE/NotificationService.swift

```diff
@@ -345,69 +351,107 @@ class NotificationService: UNNotificationServiceExtension {
                                 }
                                 
                                 if !isHighlighted {
+                                    #warning("In practice, this only hides the notification's content. An empty notification may be less useful in this instance?")
```

Comment:
This is actually some kind of notification we don't want to display at all. We'll need to use https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_usernotifications_filtering to suppress the notification. I'm not sure we can display an _empty_ notification, if we can and also desired from the design team, you can do the update.

Reply:
Is it good if I add this as a new subtask in #4132 for now, and come back to it when I next look at notifications?

Reply:
> Is it good if I add this as a new subtask in #4132 for now, and come back to it when I next look at notifications?

Yes, works for me.